tempodb/encoding/vparquet/block_search_test.go

```diff
@@ -241,7 +242,7 @@ func makeBackendBlockWithTraces(t *testing.T, trs []*Trace) *backendBlock {
 		BloomShardSizeBytes: 100 * 1024,
 	}
 
-	meta := backend.NewBlockMeta("fake", uuid.New(), VersionString, backend.EncNone, "")
+	meta := backend.NewBlockMeta("fake", uuid.New(), VersionString, backend.EncNone, model.CurrentEncoding)
```

Comment:
I think `model.CurrentEncoding` is not correct to combine with the vParquet formats. For them the dataEncoding is always blank.  The v2 block format contained different flavors of trace byte layout within the same overall block layout, so the block was a combination of version + dataEncoding.   Unless I missed a change to `model.CurrentEncoding`.

Reply:
Changed this, nice catch